### DISCUSSION TITLE
Teacher verified as student? Unable to get teacher benefits.

### DISCUSSION THREAD

**Discussion by proflivio** [community/community#77828] [GitHub Education] [answered]
### Hi everyone,

Hi,
i applied as a teacher and got approved. I couldn't enable Copilot, Codespaces and such, and then I found that the 'Your benefits' link takes me to `https://education.github.com/globalcampus/student`.
I have no idea how to correct this or who to contact about it.

Should I try and get verified again?

![githubapproved](https://github.com/community/community/assets/104489326/ef71b43f-ddac-4840-b602-c335d5b5b556)

**Comment by dakshithadissanayaka** [comment]
It is possible that you have been mistakenly verified as a student instead of a teacher. This could be due to a bug in the GitHub Education system or a clerical error.

To resolve this issue, you can contact GitHub Education support and explain the situation. They will be able to investigate the issue and correct your verification status.

Here are the steps to contact GitHub Education support:

1. Go to the GitHub Education support page: [https://education.github.com/benefits](https://education.github.com/benefits).
2. Click the **Contact us** button.
3. Select **I have a problem with my GitHub Education account** as the issue category.
4. In the **Description** field, provide a detailed explanation of the issue, including the fact that you were incorrectly verified as a student instead of a teacher.
5. Click the **Submit** button.

Once you have submitted a support ticket, a GitHub Education representative will review your request and respond as soon as possible.

In the meantime, you can try to get verified again as a teacher. However, it is important to note that GitHub Education has a limit on the number of times you can be verified. If you have already been verified twice, you will need to wait until the next verification cycle to try again.

To get verified as a teacher, you will need to provide proof of your employment at an accredited educational institution. This can be in the form of a paystub, employment contract, or school ID card. You can submit your proof of employment to GitHub Education support through the same channel that you used to contact them about the verification issue.

I hope this information is helpful. Please let me know if you have any other questions.

**Comment by proflivio** [comment, marked as answer]
> It is possible that you have been mistakenly verified as a student instead of a teacher. This could be due to a bug in the GitHub Education system or a clerical error.


Thanks, 

I've managed to contact support, and I did try verifying again, but got approved as a student for the second time.
Support has informed me (sloghtly condescendingly) that my status has been reset and try once more *carefully* selecting the teacher option.. When I tried again, a message pops out saying the proof was used before.
Thanks again, I'll continue with the support saga.

**Reply by gnaist** [reply]
This happened to me as well (I am a teacher but was verified as a student). Have you been able to be verified as a teacher through support? Or maybe I should just use the student benefits?

**Reply by bmlaw** [reply]
Me too. Anybody know of a solution?

**Reply by samus-aran** [reply]
Hi @gnaist & @bmlaw 👋 

I can help you with that. Please reply and I'll revoke your current coupon and that will allow you to re-apply as educators. 👍

**Reply by gnaist** [reply]
Yes, please! Does this mean I would go through the verification process again with my same employment status documentation?

**Reply by bmlaw** [reply]
@samus-aran Yes please.

**Reply by timwonderer** [reply]
Hi, I recently just got approved as a student too but I applied as a teacher. Can you also please revoke my current coupon so I can try again? Thank you

**Reply by alanwnl** [reply]
@samus-aran I have the same issue. I am approved, but the GitHub classroom still asks me to "Get verified to get Codespaces Education benefit."

**Reply by samus-aran** [reply]
@gnaist You can re-apply.

> Does this mean I would go through the verification process again with my same employment status documentation?

Yes, yes it does.

--------

@bmlaw & @timwonderer - Your current coupon is a Teacher's already. Is this correct?

--------

@alanwnl

> I have the same issue. I am approved, but the GitHub classroom still asks me to "Get verified to get Codespaces Education benefit."

This may be a different issue. The current coupon you have is a student, you want this revoked to re-apply as a teacher?

**Reply by flucoe** [reply]
@samus-aran this happened to me as well (teacher approved as student). Could you help me with this? Thanks.

**Reply by CarlosCuevas** [reply]
@samus-aran this also happened to me (faculty approved as student)

**Comment by zkostic** [comment]
I am a teacher as well. I have been verified in the past and I am verified currently. 
My assessment of the problem is that
- In the past several years, teachers had access to development tools
- In my case, I was able to get approved to use GitKraken client
- Recently  (end 2013), the development tools as benefits for teachers seem to have disappeared.
- Development tools are still available to students

Github support for Github education is unprofessional, it literally seems that they do not bother to check (even on a random basis) the validity of complaints.

So here we are - HELOO GIthub support - will you please check for at least one teacher - is it true that development tools are not supported any more?

**Reply by bijaykhatri23** [reply]
Yes

**Comment by CCCCSCI** [comment]
This just happened to me as well. I was very careful in selecting teacher, but was approved as a student instead.

**Reply by timwonderer** [reply]
The whole approval process is a hot mess with the support team being no better than giving condescending answers

**Comment by viktor-vad** [comment]
Same here. i created a ticket about it, and got an automatic reply (not useful at all), and the ticket got closed. Problem NOT solved.

**Reply by samus-aran** [reply]
Hi @viktor-vad you got an automatic reply because you selected an option via the support form that was not related to revoking your discount coupon. We can see you have been answered by our Support team ❤️

**Comment by wrogner** [comment]
I have the same issue.
@Github: please remedy and grant teacher access.
Thanks

**Reply by samus-aran** [reply]
Hi @wrogner 

These types of requests we can help you on the Community. Mention myself for quicker response 👍 We can not grant you your preferred type of coupon ourselves, we can only revoke and ask that you re-apply (hopefully obtaining the alternative coupon 😄)

You may re-apply now.

**Reply by wrogner** [reply]
Oh thanks, now I know why Copilot suddenly stopped working

**Reply by mrstaylormags** [reply]
@samus-aran I am having the same issue! Can you please reset my status so I can reapply for verified educator status?

**Comment by juliaSchedler** [comment]
This just happened to me too! I used to be a student at the institution I currently work at, so I'm wondering if that has anything to do with it.

**Reply by samus-aran** [reply]
Hi @juliaSchedler 👋  

Your academic status has been updated. Please visit [GitHub Education](https://education.github.com/discount_requests/application) to submit a new request to verify your academic status as a teacher.
 
You may need to provide a new image capture of your academic proof taken directly with your device camera.

Hope that helps 👍

**Comment by mayankgituni** [comment]
Hi @samus-aran, Do you think you can help with my account issue?

I have be approved for the Github Teams through the teacher program but I can not see any feature enabled. My account is still set under the Free account.

![Screenshot from 2024-03-14 14-20-45](https://github.com/community/community/assets/43289219/2f06d630-30df-43c1-800c-832d88996166)

I am a time crunch as I have a class of students waiting on the account to work. Can you help me? When can i expect the features to be enabled?

**Comment by CarlosCuevas** [comment]
@samus-aran i'm also in this situation

**Comment by flucoe** [comment]
Hi @samus-aran I'm also in this situation (teacher approved as student). Could you help me with this? Thanks.

**Comment by chrisdonahue** [comment]
Hi @samus-aran I am also in this situation. I've been repeatedly verified as a student despite applying as a teacher. Could my academic status be reset so I can try the application again as a teacher?

Additional context: I suspect this glitch is because I was previously verified as a student for GitHub Education, but I have recently transitioned to a faculty role. I am quite certain I applied as a teacher, not a student.

**Comment by Ashish-Subramanian** [comment]
Hi @samus-aran, I'm also in the same situation! I'm certain I applied as a teacher, so I'm not sure what the issue is, but my proof clearly shows that I am an employee of the university and not a student. What should I change upon reapplying to ensure this doesn't happen again?

**Comment by d2kriege** [comment]
Anything new on this issue? applied (most definitely) as a teacher, approved as a student, and have no access to teacher's benefits! Please help!

**Reply by samus-aran** [reply]
Hi @d2kriege, we can confirm that you are verified as a Teacher on our system. Can you explain what teacher's benefits are you not able to access?

**Comment by ibrahimTurkekul** [comment]
I work as an Information Technologies Teacher at a vocational high school (Istanbul/Turkey). None of the documents I uploaded to GitHub were accepted. I sent my teacher identity, the document of my workplace, and a photo of the Ministry of Education module screen, but I failed with all of them. How can I register as a teacher? Can someone help?"

**Reply by samus-aran** [reply]
@ibrahimTurkekul I suspect it's the image quality.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.


1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.

**Comment by BparejaTec** [comment]
Hi @samus-aran  I'm also in this situation (teacher approved as student). Could you help me with this? Thanks!

**Comment by adririquelme** [comment]
Hi @samus-aran,

I seem to be in a very similar situation.

I am currently verified as a Student and my GitHub Education benefits are active until July 2028. However, I am currently a faculty member and researcher at the University of Alicante (Spain), and I would like to apply for Teacher verification instead.

My institutional email address (@ua.es) is already verified on GitHub, but the Education Benefits page shows an active Student coupon and the "Start an application" button is disabled, so I cannot submit a Teacher application.

Would it be possible to review my academic status and, if necessary, revoke/reset my current Student coupon so that I can re-apply as a Teacher?

GitHub username: adriquelme

Thank you very much for your help.
